cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -62,6 +62,21 @@
         <span class="tip tip-stacked timezone">(UTC)</span>
       </div>
     </li>
+    <li>
+        <label id="heading-entrance-exam">${_("Entrance Exam")}</label>
+            <div class="show-data">
+                <div class="heading">
+                    <input type="checkbox" id="entrance-exam-enabled" />
+                    <label for="entrance-exam-enabled">${_("Enables entrance exam requirement. Students must successfully complete exam before gaining access to the course.")}</label>
+                </div>
+                <div id="div-grade-requirements">
+                    <p><span class="tip tip-inline">${_("You can now view and author your course entrance exam from the ")}<a href='${course_handler_url}'>${_("Course Outline")}</a></span></p>
+                    <p><label>${_("Grade Requirements")}</label></p>
+                    <p><div><input type="text" id="entrance-exam-minimum-score-pct"><span class="tip tip-inline">${_("as a percent, e.g. 50")}</span></div></p>
+                    <p><span class="tip tip-inline">${_("The score student must meet in order to successfully complete the entrance exam. ")}</span></p>
```

Comment:
This text should also be programmatically associated with that form field.  luckily aria-describedby takes a space-separated list of IDrefs:
```<input type="text" id="entrance-exam-minimum-score-pct" aria-describedby="min-score-format min-score-tip"><span id="min-score-format" class="tip tip-inline">${_("as a percent, e.g. 50")}</span>```
...
```<span class="tip tip-inline" id="min-score-tip">${_("The score student must meet in order to successfully complete the entrance exam. ")}</span>```